galsim/base.py

```diff
@@ -1316,15 +1465,19 @@ def drawImage(self, image=None, nx=None, ny=None, bounds=None, scale=None, wcs=N
         if method == 'sb':
             gain *= local_wcs.pixelArea()
 
+        # Account for area and exptime.
+        prof *= area * exptime
+
         # Making a view of the image lets us change the center without messing up the original.
         imview = image.view()
         imview.setCenter(0,0)
 
         if method == 'phot':
             try:
-                image.added_flux = prof.SBProfile.drawShoot(
+                added_photons = prof.SBProfile.drawShoot(
                     imview.image, n_photons, uniform_deviate, gain, max_extra_noise,
                     poisson_flux, add_to_image)
+                image.added_flux = added_photons / area / exptime
```

Comment:
Good point.  I added the following to the drawImage() docstring just after the discussion about `.added_flux`: 

```
        Also, recall that the relationship between flux (in photons/cm^2/s) and the pixel values
        depends on `exptime`, `area`, `gain`, and `method`!  For example:

            >>> obj = galsim.Gaussian(fwhm=1)
            >>> obj.flux
            1.0
            >>> im = obj.drawImage()
            >>> im.added_flux
            0.9999630988657515
            >>> im.array.sum()
            0.99996305
            >>> im = obj.drawImage(exptime=10, area=10)
            >>> im.added_flux
            0.9999630988657525
            >>> im.array.sum()
            99.996315
            >>> im = obj.drawImage(exptime=10, area=10, method='sb', scale=0.5, nx=10, ny=10)
            >>> im.added_flux
            0.9999973790505298
            >>> im.array.sum()
            399.9989
            >>> im = obj.drawImage(exptime=10, area=10, gain=2)
            >>> im.added_flux
            0.9999630988657525
            >>> im.array.sum()
            49.998158
```

I think this, plus the discussion at the end of the different drawImage methods "To convert from flux (photons/cm^2/s) into image values of analog-to-digital units (ADU) ..." may be enough?  Let me know if you think more clarification would be helpful.